index.d.ts

```diff
@@ -204,3 +204,23 @@ export type RequireAtLeastOne<ObjectType, KeysType extends keyof ObjectType = ke
 	}[KeysType]
 	// …then, make intersection types by adding the remaining properties to each mapped type.
 	& Omit<ObjectType, KeysType>;
+
+/*
+Converts a readonly object into a mutable object. Inverse of Readonly<T>.
+
+@example
+```
+import {Mutable} from 'type-fest';
+
+type Foo = {
+	readonly a: number;
+	readonly b: string;
+};
+
+const mutableFoo: Mutable<Foo> = {a: 1, b: '2'};
+mutableFoo.a = 3;
+```
+*/
+export type Mutable<ObjectType> = {
+	-readonly [KeyType in keyof ObjectType]: ObjectType[KeyType];
```

Comment:
Would be useful for learning purposes for users if there was an inline explanation of how it works, like https://github.com/sindresorhus/type-fest/blob/d84eedf0178aa98056cc03f7851dfae39f7fc90c/index.d.ts#L197-L206